### ISSUE TITLE
Problem: C4Social process needs a synopsis and introductory text

### ISSUE THREAD

**Issue by aschrijver** [c4social/C4-tools#13]
The README mentions:

> "To me it sounded perfect for FLOSS so I wondered why the process not more widely adopted. My best guess is that people just haven't tried it."

I think I can list some reasons. The [C4 Spec](https://rfc.zeromq.org/spec/42/) has a reasonable Abstract and an appealing set of Goals, but is otherwise a quite formal specification full of SHALL and MUST bullet points of which the implications aren't really clear unless one takes the time to study them thoroughly. I a similar vein the [C4 Process documentation](https://zguide.zeromq.org/docs/chapter6/#The-ZeroMQ-Process-C) at the ZeroMQ website is a wall of text, that takes similar amount of time and attention to parse.

What is needed IMHO is a good synopsis and introductory text for noobs that can be consumed really quickly. A cheat-sheet if you will, and maybe having some process diagrams as well. In other words, some kind of article with 4-5 mins of read-time, after which one says "I really NEED this".

**Comment by aschrijver** [comment]
In addition to the above I'll describe my thought process when I first encountered C4, mentioned by @weex on the [Open Hospitality Network](https://openhospitality.network) matrix room:

> [opens link, starts a quickscan] "Goals... MUST, SHALL, branch, patch, MPL/GPL... details, details, yada, yada, yada... Okay, interesting to revisit once I get to technical process elaboration in my planned projects." --> Bookmark-And-Forget

Or rather: Adds to a humongous pile of Very-interesting-stuff™ which I'll _surely_ remember when the need arises :thinking: 
This is how things work for many people, I think. Time is precious, and everything is vying for attention.

Other than that, another reason people may skip C4 process in their project, is simply because they start on a KISS + YAGNI basis. Only when upfront you have much higher ambitions for your project, will most people deep-dive the spec.

**Comment by weex** [comment]
I agree this is a problem worth solving as currently we don't have good material for newcomers to quickly grok the process and the problems it solves.

**Comment by weex** [comment]
Made a couple of flowcharts that need a slight bit of editing but then can be dropped in a quick pitch.

![Standard PR Process (1)](https://user-images.githubusercontent.com/38776/131301124-785cd978-3c25-4b92-9b31-a0002a09b9f1.png)
![C4 Process](https://user-images.githubusercontent.com/38776/131301136-6e042c8a-bf0f-4936-b460-b9a283ddf656.png)
